.scripts/prepare_workflow.py

```
@@ -4,97 +4,153 @@
-  - delay: {wait_time_after_release}ms
-  """
+    return (
+        "  - set-control:\n"
+        f"      part-id: {button_name}\n"
+        "      control: pressed\n"
+        "      value: 0\n"
+        f"  - delay: {wait_time_after_release}ms\n"
+    )
+
 
 def replace_lines(file_path, compiled_replacements):
```

Ruff (0.15.7)

[warning] Too many branches (14 > 12)

(PLR0912)